validator.py

```diff
@@ -32,7 +48,18 @@ def get_all_defined_messages() -> list:
     definition_message_names = get_all_defined_messages()
     print(f"all definition message names requested = {definition_message_names}")
 
+    requested_can_line_names = get_all_requested_can()
+    print(f"all requested can line names = {requested_can_line_names}")
+
+    defined_can_names = get_all_defined_can()
+    print(f"all defined can line names = {defined_can_names}")
+
     for definition_message_name in definition_message_names:
         if definition_message_name not in can_message_names:
```

Comment:
Shouldn't you also check that all can messages have a definitions.yml entry?

Reply:
I want to make it so that we can deprecate a message but still have its definition so that we could remove a part of the car and not have to write the message type when we re-add it to the car